server/neptune/workflows/internal/terraform/job/init_step_runner.go

```diff
@@ -0,0 +1,33 @@
+package job
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/activities"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/job"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/root"
+	"go.temporal.io/sdk/workflow"
+)
+
+type initActivities interface {
+	TerraformInit(ctx context.Context, request activities.TerraformInitRequest) (activities.TerraformInitResponse, error)
+}
+
+type InitStepRunner struct {
+	Activity initActivities
+}
+
+func (r *InitStepRunner) Run(executionContext *job.ExecutionContext, localRoot *root.LocalRoot, step job.Step) (string, error) {
+	var resp activities.TerraformInitResponse
+	err := workflow.ExecuteActivity(executionContext.Context, r.Activity.TerraformInit, activities.TerraformInitRequest{
+		Step:      step,
+		Envs:      executionContext.Envs,
+		TfVersion: executionContext.TfVersion,
```

Comment:
can you pass in the terraform version into the activity? I think it currently uses the local root, which doesn't assign that field yet. https://github.com/lyft/atlantis/blob/4da3ccf194c53ed14aaa26e9712c62aff4263d39/server/neptune/workflows/internal/terraform/job/job_runner.go#L42